lib/modules/apostrophe-pieces/lib/api.js

```diff
@@ -395,7 +394,7 @@ module.exports = function(self, options) {
         if (err) {
           return callback(err);
         }
-        return self.apos.docs.deduplicateTrash(req, piece, callback);
+        return self.apos.docs.getManager(self.name).deduplicateTrash(req, piece, callback);
```

Comment:
This is redundant, this module is the manager of its own type, so just say self.deduplicateTrash.